src/config.rs

```diff
@@ -76,6 +80,31 @@ impl Config {
     }
 }
 
+fn manual_override(mut toml: String, node_url: Option<String>, db_url: Option<String>) -> String {
```

Comment:
Brittle but should get the job done. Couldn't we use `let mut config = toml::from_str(&toml)?;` and then override from environment as necessary?

Reply:
That is a really nice suggestion! However, this would require adding a "set_field" method to those private attributes (as long as we're ok with that). I think your suggestion is MUCH better than the current approach. I will do that.

Reply:
I have made the suggested changes (it all looks much better!)